src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -169,6 +169,184 @@
+ * </pre>
+ * <p>
+ * To configure the check to include only "com" packages as third party group imports
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;CustomImportOrder&quot;&gt;
+ *   &lt;property name=&quot;customImportOrderRules&quot;
+ *     value=&quot;STATIC###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE&quot;/&gt;
+ *   &lt;property name=&quot;thirdPartyPackageRegExp&quot; value=&quot;^com\.&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:-
+ * </p>
+ * <pre>
+ * package com.company;
+ *
+ * import static java.util.*; //OK
+ * import static java.io.*; //OK
+ *
+ * import java.time.*; //OK
+ * import javax.net.*; //OK
+ *
+ * import org.apache.commons.io.FileUtils; //violation(should be in end)
+ * import com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck; //violation
+ * import com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck; //OK
+ * </pre>
+ * <p>
+ * To configure the check to include "com" packages as third party packages
+ * and "org" packages in special import group
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;CustomImportOrder&quot;&gt;
+ *   &lt;property name=&quot;customImportOrderRules&quot;
+ *     value=&quot;STATIC###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE&quot;/&gt;
+ *   &lt;property name=&quot;specialImportsRegExp&quot; value=&quot;^org\.&quot;/&gt;
+ *   &lt;property name=&quot;thirdPartyPackageRegExp&quot; value=&quot;^com\.&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:-
+ * </p>
+ * <pre>
+ * package com.company;
+ *
+ * import static java.util.*; //OK
+ * import static java.io.*; //OK
+ *
+ * import java.time.*; //OK
+ * import javax.net.*; //OK
+ * import org.apache.commons.io.FileUtils; //OK
+ * import com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck; //violation
+ * import com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck; //OK
+ * </pre>
+ * <p>
+ * To configure the check such that empty line separator between two groups is disabled:-
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;CustomImportOrder&quot;&gt;
+ *   &lt;property name=&quot;customImportOrderRules&quot;
+ *     value=&quot;STATIC###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE&quot;/&gt;
+ *   &lt;property name=&quot;specialImportsRegExp&quot; value=&quot;^org\.&quot;/&gt;
+ *   &lt;property name=&quot;thirdPartyPackageRegExp&quot; value=&quot;^com\.&quot;/&gt;
+ *   &lt;property name=&quot;separateLineBetweenGroups&quot; value=&quot;false&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example:-
+ * </p>
+ * <pre>
+ * package com.company;
+ *
+ * import static java.util.*; //OK
+ * import static java.io.*; //OK
+ *
+ * import java.time.*; //OK
+ * import javax.net.*; //OK
+ * import org.apache.commons.io.FileUtils; //OK
+ * import com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck; //OK
+ * import com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck; //OK
```

Comment:
please keep some violation in code to let user see a difference, when all is "OK" it is not clear what code is not ok.

please apply this guideline to other examples too

Reply:
Okay...I will make these changes as soon as possible.